src/codemagic/firebase/resources/release.py

```diff
@@ -0,0 +1,34 @@
+from dataclasses import dataclass
+from dataclasses import field
+from datetime import datetime
+from datetime import timezone
+
+from .abstract_base_resource import AbstractFirebaseResource
+
+
+@dataclass
+class FirebaseReleaseResource(AbstractFirebaseResource):
+    """https://firebase.google.com/docs/reference/app-distribution/rest/v1/projects.apps.releases#ReleaseNotes
+    """
```

Comment:
Usually we use this kind of formatting:
```suggestion
    """
    https://firebase.google.com/docs/reference/app-distribution/rest/v1/projects.apps.releases#ReleaseNotes
    """
```